dev/test.clj

```diff
@@ -338,6 +340,129 @@
       (is (= {:type "FILE" :code "添付ファイル" :label "添付ファイル"}
              (select-keys file-field test-fields))))))
 
+(comment
```

Comment:
I don't know why these tests are commented out, but these test will work. I'll check it later.

Reply:
These are commented out because these tests can cause the deletion of user (by @egs33 )